objects/screen.c

```diff
@@ -1016,6 +1016,57 @@ luaA_screen_count(lua_State *L)
     return 1;
 }
 
+/** Add a fake screen.
+ * @tparam integer x X-coordinate for screen.
+ * @tparam integer y Y-coordinate for screen.
+ * @tparam integer width width for screen.
+ * @tparam integer height height for screen.
+ * @return The new screen.
+ * @function fake_add
+ */
+static int
+luaA_screen_fake_add(lua_State *L)
```

Comment:
Yup, that's planned, but only after "just disabling" screens works. One problem at a time. :-)